velox/core/ITypedExpr.h

```diff
@@ -81,7 +81,11 @@ class ITypedExpr {
 
   virtual bool operator==(const ITypedExpr& other) const = 0;
 
+  static void registerSerDe();
+
  protected:
+  folly::dynamic serializeBase(std::string name) const;
```

Comment:
in most cases you pass a string literal ("ConcatTypedExpr"), so you wouldn't really need to materialize an std::string here (maybe take std::string_view instead?). But not a big deal.